app/controllers/partners/base_controller.rb

```diff
@@ -10,5 +10,32 @@ class BaseController < ApplicationController
     def redirect_to_root
       redirect_to root_path
     end
+
+    helper_method :current_partner
+    def current_partner
+      current_partner_user.partner
+    end
+
+    helper_method :valid_items
+    def valid_items
+      # @valid_items ||= DiaperBankClient.get_available_items(current_partner.diaper_bank_id)
+      @valid_items ||= current_partner.organization.valid_items
+    end
+
+    helper_method :item_id_to_display_string_map
+    def item_id_to_display_string_map
+      @item_id_to_display_string_map ||= valid_items.each_with_object({}) do |item, hash|
+        # hash[item["id"].to_i] = item["name"]
+        hash[item[:id].to_i] = item[:name]
+      end
+    end
+
+    # Copied from partner's application_helper.rb
+    # https://github.com/rubyforgood/partner/blob/c597ca849f0074d3b8a458e1950d19c4b3e6f4d0/app/helpers/application_helper.rb#L10-L12I
+    helper_method :valid_items_for_select
+    def valid_items_for_select(items)
+      # items.map { |item| [item["name"], item["id"]] }.sort
+      items.map { |item| [item[:name], item[:id]] }.sort
+    end
```

Comment:
I feel like these may make sense to be instance methods on `Organization`? And call them only when needed in the specified controller.

Thoughts?

Reply:
yeah good idea. I'll move it there